Plugins/DeleteUnusedBranches/DeleteUnusedBranchesForm.cs

```diff
@@ -125,21 +127,26 @@ private static IEnumerable<string> GetObsoleteBranchNames(RefreshContext context
 
             var args = new GitArgumentBuilder("branch")
             {
+                 "--list",
                  { context.IncludeRemotes, "-r" },
                  { !context.IncludeUnmerged, "--merged" },
                  context.ReferenceBranch
             };
 
-            string[] branches = context.Commands.GitExecutable.GetOutput(args)
-                .Split('\n')
-                .Where(branchName => !string.IsNullOrEmpty(branchName))
-                .Select(branchName => branchName.Trim('*', ' ', '\n', '\r'))
-                .Where(branchName => branchName != "HEAD" && branchName != curBranch &&
-                                   branchName != context.ReferenceBranch)
-                .ToArray();
+            var result = context.Commands.GitExecutable.Execute(args);
 
-            return branches.Where(branchName => (!context.IncludeRemotes || branchName.StartsWith(context.RemoteRepositoryName + "/")) &&
-                                   (regex == null || regex.IsMatch(branchName) == regexMustMatch)).ToArray();
+            if (result.ExitedSuccessfully)
+            {
+                return _commandOutputParser.GetBranchNames(result.AllOutput)
+                                           .Where(branchName => branchName != curBranch && branchName != context.ReferenceBranch)
+                                           .Where(branchName => (!context.IncludeRemotes || branchName.StartsWith(context.RemoteRepositoryName + "/")) && (regex == null || regex.IsMatch(branchName) == regexMustMatch))
+                                           .ToArray();
+            }
+            else
```

Comment:
`else` is redundant.

Purely as an observation, I prefer to structure the code to have the "happy path" exit as the very last statement. This typically makes the code easier to follow.